src/helm/common/images_utils.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import io
+import requests
+import shutil
+from typing import Optional
+
+from PIL import Image
+
+from .general import is_url
+
+
+def open_image(image_location: str) -> Image:
+    """
+    Opens image with the Python Imaging Library.
+    """
+    image: Image
+    if is_url(image_location):
+        image = Image.open(requests.get(image_location, stream=True).raw)
+    else:
+        image = Image.open(image_location)
+    return image.convert("RGB")
+
+
+def encode_base64(image_location: str) -> str:
+    """Returns the base64 representation of an image file."""
+    image_file = io.BytesIO()
+    image: Image = open_image(image_location)
+    image.save(image_file, format="JPEG")
```

Comment:
1. Consider taking format as a parameter
2. Consider using PNG, which is usually lossless (JPEG is usually lossy).